lb/Consolidate gias school and school models (#90)
DFE-Digital/itt-mentor-services: docs/data-model.md

### Schools

> All schools from the gias import will have a record in the `schools` table (the `School` entity in our ERD).

Reviewer: Let's try to stylise it as GIAS wherever possible. That's what's used on [the service's about page](https://get-information-schools.service.gov.uk/about).

```suggestion
All schools from the GIAS import will have a record in the `schools` table (the `School` entity in our ERD).
```

> The `placements` and `claims` boolean attributes will indicate which service(s) the School has been onboarded into. It's possible for Schools to be onboarded to only one service, or both.

Reviewer: ```suggestion
The `placements` and `claims` boolean attributes will indicate which service(s) the School has been onboarded into. It's possible for Schools to be onboarded to both services, one service, or neither service.
```

For example:

| urn    | name     | placements | claims |
|--------|----------|------------|--------|
| 100000 | School A | 1          | 0      |
| 100001 | School B | 0          | 1      |
| 100002 | School C | 1          | 1      |
| 100003 | School D | 0          | 0      |

- School A has only been onboarded into the School Placements service.
- School B has only been onboarded into the Track & Pay service.
- School C has been onboarded into both services.

> - School D is a school from the Gias import, but it is not onboarded into either service

Reviewer: ```suggestion
- School D is a school from the GIAS import, but it is not onboarded into either service
```

> If a School doesn't exist in the `schools` table, it hasn't been onboarded to either service yet.
> 

Reviewer: ```suggestion
```
